api/envoy/api/v2/route/route_components.proto

```diff
@@ -875,6 +875,16 @@ message RouteAction {
 
   InternalRedirectAction internal_redirect_action = 26;
 
+  // An internal redirect is handled, iff the number of previous internal redirects that a
+  // downstream request has encountered is lower than this value, and internal_redirect_action
+  // is set to HANDLE_INTERNAL_REDIRECT.
+  // In the case of a downstream request is bounced among multiple routes by internal redirect,
```

Comment:
In the case where a downstream request is bounced..

Reply:
Done.